app/components/teaching_events/event_component.rb

```diff
@@ -0,0 +1,60 @@
+module TeachingEvents
+  class EventComponent < ViewComponent::Base
+    attr_reader :title, :type, :slug
+
+    def initialize(event:)
+      @event     = event
+      @title     = event.name
+      @type      = event.type_id
+      @online    = event.is_online
+      @virtual   = event.is_virtual
+      @in_person = event.building.present?
+      @start_at  = event.start_at
+      @end_at    = event.end_at
+      @slug      = event.readable_id
+
+      super
+    end
+
+    # check if event is 'train to teach' or 'question time'
+    def train_to_teach?
+      type.in?([222_750_001, 222_750_007])
```

Comment:
Worth references the constants here and other places they are hard coded?

https://github.com/DFE-Digital/get-into-teaching-app/blob/master/lib/extensions/get_into_teaching_api_client/constants.rb#L3-L9